test/wallet-http-test.js

```diff
@@ -252,13 +252,14 @@ describe('Wallet HTTP', function() {
   });
 
   it('should create an open and broadcast the tx', async () => {
+    let entered = false;
+    const handler = () => entered = true;
```

Comment:
This comment applies here:
https://github.com/handshake-org/hsd/pull/537/files#r559782100

Reply:
I think its worth adding test coverage here that it is only called once to prevent applications that rely on the websocket from getting the same notification twice. I don't think that is happening but I don't think any sort of test exists yet that covers that particular case

Reply:
Ok interesting idea, so instead of using `once` maybe we'd replace `entered` with a counter and at the end of the test, assert the counter was at `1` ?

Reply:
> Ok interesting idea, so instead of using once maybe we'd replace entered with a counter and at the end of the test, assert the counter was at 1 ?

Yup that would work great. Will approve this PR after that change

Reply:
added in c36f27026c3dd6322904d4ae8ba20adbe02417a3